frontend/apps/core-app/src/screens/RecipientListScreen/__tests__/RecipientListScreen.component.test.tsx

```diff
@@ -0,0 +1,21 @@
+import { NavigationProp, useNavigation } from '@react-navigation/native';
+import React from 'react';
+
+import { render } from '../../../testUtils/render';
+import { RecipientListScreenComponent } from '../RecipientListScreen.component';
+import { RootParamList } from '../../../navigators/types';
+
+it('should match the snapshot', () => {
+  const Wrapper: React.FC = () => {
+    const navigation = useNavigation<NavigationProp<RootParamList>>();
+    return (
+      <RecipientListScreenComponent
+        route={{ name: 'RecipientList', params: {}, key: 'key' }}
+        navigation={navigation}
+      />
```

Comment:
To test this I wouldn't normally use the wrapper/hook.

This will be easier to test for any future tests
```
const navigateMock = jest.fn();
const { toJSON } = render(<RecipientListScreenComponent
  route={{ name: 'RecipientList', params: {}, key: 'key' }}
  navigation={{ navigate: navigateMock }}
/>);
```